ibis/backends/flink/identifiers.py

```diff
@@ -0,0 +1,560 @@
+from __future__ import annotations
+
+# https://nightlies.apache.org/flink/flink-docs-release-1.18/docs/dev/table/sql/overview/#reserved-keywords
```

Comment:
I couldn't find anywhere in the code to pull this from programmatically. 🤷

Reply:
Instead of maintaining this list, why don't we quote every indentifier?

Reply:
> Instead of maintaining this list, why don't we quote every identifier?

Done, as discussed.